com.vrcfury.vrcfury/Editor/VF/Service/ActionClipService.cs

```diff
@@ -368,6 +368,25 @@ public class BuiltAction {
                 .Select(t => t.Value)
                 .DefaultIfEmpty(ShaderUtil.ShaderPropertyType.Float)
                 .First();
+
+            // this is a list of all distinct shader keywords for the materials

```

Comment:
This method is called rapidly, including within the UI of the material editor. Printing to the debug log is probably not a good solution here. This would be more reasonable as a visible warning in the VRCFuryActionEditor, where it could warn if the property you've selected is a keyword.